app/src/main/java/com/fastlink/zekrapp/appData/data/Zekrs.kt

```diff
@@ -9,10 +10,10 @@ fun getZekrs(){
             ZekrModel(
                 id = i[0].toString().toInt(),
                 categoryId = i[1].toString().toInt(),
-                description = i[2].toString(),
-                descriptionAbstract = i[3].toString(),
+                zekirTitle = i[2].toString(),
+                plainZekirTitle = i[3].toString(),
                 hint = i[4].toString(),
-                counter = i[5].toString(),
+                counterAsString = i[5].toString(),
```

Comment:
Counter is enough, no need to add asString to it

Reply:
We have two counters for  zikr recitation: one counter records the number of recitations numerically, while the other counter represents the number  as a textual string
for example 
this is counterASString Varibal
<img width="359" alt="Screenshot 2024-08-15 at 2 17 35 PM" src="https://github.com/user-attachments/assets/dc0b47a9-10d5-4394-a900-ca62e232dc4a">

and this is counter 
<img width="417" alt="Screenshot 2024-08-15 at 2 18 18 PM" src="https://github.com/user-attachments/assets/6c4b2392-0597-4561-9e87-3360fdb247c7">

Reply:
But numbers can be stored in a string too, name them _numericalCounter_ and _textCounter_.